### DISCUSSION TITLE
file reading failure

### DISCUSSION THREAD

**Discussion by lyondns** [DHI/mikeio1d#125] [Q&A]
I have a file .res1d from mike 2014 .when I use pycharm (python 3.12, mikeio1d 0.8.2) to read it, I failed.

> from mikeio1d import Res1D
> res = Res1D("C:\\Users\\liuyan\\Desktop\\test\\asd.res1d")
> df = res.read()

 the tips

> D:\python3.12.0\python.exe D:/test/read.py
> Traceback (most recent call last):
>   File "D:\test\read.py", line 3, in <module>
>     res = Res1D("C:\\Users\\liuyan\\Desktop\\test\\asd.res1d")
>           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "D:\python3.12.0\Lib\site-packages\mikeio1d\res1d.py", line 135, in __init__
>     self.result_network = ResultNetwork(self)
>                           ^^^^^^^^^^^^^^^^^^^
>   File "D:\python3.12.0\Lib\site-packages\mikeio1d\result_network\result_network.py", line 83, in __init__
>     self.set_result_locations()
>   File "D:\python3.12.0\Lib\site-packages\mikeio1d\result_network\result_network.py", line 119, in set_result_locations
>     self.structures = ResultStructures(res1d)
>                       ^^^^^^^^^^^^^^^^^^^^^^^
>   File "D:\python3.12.0\Lib\site-packages\mikeio1d\result_network\result_structures.py", line 35, in __init__
>     self.set_structures()
>   File "D:\python3.12.0\Lib\site-packages\mikeio1d\result_network\result_structures.py", line 50, in set_structures
>     result_structure_attribute_string = make_proper_variable_name(
>                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "D:\python3.12.0\Lib\site-packages\mikeio1d\result_network\various.py", line 41, in make_proper_variable_name
>     string = string.translate(_translator_table)
>              ^^^^^^^^^^^^^^^^
> AttributeError: 'NoneType' object has no attribute 'translate'

**Comment by ryan-kipawa** [comment]
Hi @lyondns 
Would it be possible to supply such a file? In general we don't aim to support such old files, but if it's an easy fix, we can try to roll it in.

**Reply by lyondns** [reply]
[Uploading 11.ADDOUT等2个文件.zip…]()

**Reply by ryan-kipawa** [reply]
I don't think that worked. You need to include only the result file, no zip. Probably security related

**Reply by lyondns** [reply]
[Uploading 11.res1d…]()

**Reply by ryan-kipawa** [reply]
Unfortunately this is also not working. Please verify you're able to download the file from GitHub before we look into this.

**Reply by lyondns** [reply]
Sorry, I am wrong. I opened another .res11 file and read normally.
